app/src/main/java/com/appunite/loudius/di/NetworkModule.kt

```diff
@@ -100,4 +101,7 @@ object NetworkModule {
         GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
             .registerTypeAdapter(LocalDateTime::class.java, LocalDateTimeDeserializer())
             .create()
+
+    @Provides
```

Comment:
suggestion: it's redundant, because you have `@Inject` annotation in `ApiRequester` and it doesn't implement any interfaces

Reply:
Thanks for that 👌  Firstly I got the error with not resolved deps so I've added provides.